From b241709a0dec69349a7aef726477eacfc1bd90f6 Mon Sep 17 00:00:00 2001
From: Owen O'Malley <omalley@apache.org>
Date: Wed, 21 Sep 2011 20:30:55 +0000
Subject: [PATCH 131/795] MR1: HADOOP-7644. Add forgotten service provider file.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-0.20-security@1173833 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 240940f4cb9b657b1ed92cf7e913bb140f885964)
(cherry picked from commit d8cb7da2786978214b03ecef4a4589f34501cfb8)
(cherry picked from commit 459acc648982f7e60cd5b432cb2da40d1caa1283)
---
 .../org.apache.hadoop.security.token.TokenRenewer  |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)
 create mode 100644 src/test/META-INF/services/org.apache.hadoop.security.token.TokenRenewer

diff --git a/src/test/META-INF/services/org.apache.hadoop.security.token.TokenRenewer b/src/test/META-INF/services/org.apache.hadoop.security.token.TokenRenewer
new file mode 100644
index 0000000..b39f5a9
--- /dev/null
+++ b/src/test/META-INF/services/org.apache.hadoop.security.token.TokenRenewer
@@ -0,0 +1,2 @@
+org.apache.hadoop.tools.TestDelegationTokenFetcher$FakeRenewer
+org.apache.hadoop.mapreduce.security.token.TestDelegationTokenRenewal$Renewer
-- 
1.7.0.4

